pkg/cortex/runtime_config.go

```diff
@@ -24,10 +29,18 @@ func loadRuntimeConfig(r io.Reader) (interface{}, error) {
 
 	decoder := yaml.NewDecoder(r)
 	decoder.SetStrict(true)
-	if err := decoder.Decode(&overrides); err != nil {
+
+	// Decode the first document. An empty document (EOF) is OK.
+	err := decoder.Decode(&overrides)
+	if err != nil && !errors.Is(err, io.EOF) {
```

Comment:
```suggestion
	if err := decoder.Decode(&overrides); err != nil && !errors.Is(err, io.EOF) {
```

(for "consistency" with next `if`)